Bluepill-cli/Bluepill-cli/Bluepill/Bluepill.m

```diff
@@ -231,6 +241,33 @@ - (void)createSimulatorWithContext:(BPExecutionContext *)context {
     [context.runner createSimulatorWithDeviceName:deviceName completion:handler.defaultHandlerBlock];
 }
 
+- (void)reuseSimulatorWithContext:(BPExecutionContext *)context {
+    NSString *stepName = REUSE_SIMULATOR(context.attemptNumber);
+    
+    [[BPStats sharedStats] startTimer:stepName];
+    [BPUtils printInfo:INFO withString:stepName];
+    
+    if ([context.runner useSimulatorWithDeviceID: [[NSUUID alloc] initWithUUIDString:context.config.useDeviceID]]) {
+        context.simulatorCreated = YES; //if we don't set this flag, deleteSimulatorWithContext() won't proceed
+        
+        NEXT([self installApplicationWithContext:context]);
```

Comment:
we'd better erase the device before installing the app again. Not sure if Apple will update the existing app (keep resource/data) or remove and reinstall the app